### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by AlvaroVega**
This fix was introduced by https://github.com/telefonicaid/fiware-sth-comet/pull/515 (incluced in 2.7.0)
but was deleted  by https://github.com/telefonicaid/fiware-sth-comet/pull/536 (included in 2.8.0)

issue https://github.com/telefonicaid/fiware-sth-comet/issues/514

### CODE REVIEW

**Comment by fgalan** [comment]
Maybe the CNR entry in PR https://github.com/telefonicaid/fiware-sth-comet/pull/515 should be recovered

```
 - Ensure permissions over temp directory (used by CSV generation) in docker container (issue re-introduced in 2.8.0)
```

File: docker/Dockerfile [hunk 1]
```diff
@@ -86,6 +86,9 @@ RUN \
 	apt-get install -y git && \
 	echo "INFO: npm install --production..." && \
 	npm install --production && \
+	# Ensure permissions over default directory used by STH to generate csv files
+        mkdir temp && \
+        chmod 777 temp && \
```
Comment (fgalan) [inline docker/Dockerfile:91]: Indent is weird... maybe some [tab vs space](https://www.youtube.com/watch?v=SsoOG6ZeyUI) issue :)

Comment (AlvaroVega) [reply docker/Dockerfile]: fixed in https://github.com/telefonicaid/fiware-sth-comet/pull/562/commits/6407409a8b529b1ad0691455913742d787f48853

**Comment by AlvaroVega** [comment]
> Maybe the CNR entry in PR #515 should be recovered
> 
> ```
>  - Ensure permissions over temp directory (used by CSV generation) in docker container (issue re-introduced in 2.8.0)
> ```

Done in https://github.com/telefonicaid/fiware-sth-comet/pull/562/commits/9dcebd51a0644d994810e91b9baae54c30b45ee5

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**